main.go

```diff
@@ -93,11 +94,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	secretLabel := os.Getenv("AUTHORINO_SECRET_LABEL")
```

Comment:
We have a func to fetch env vars with default value (see line 147).

Reply:
Ok, just take into account that if the value is set, but empty, LookupEnv will return Ok. I can reuse this function if you want.